rep-0145.rst

```diff
@@ -93,26 +93,50 @@ Topics
 
```

Comment:
https://github.com/ros-infrastructure/rep/pull/186#discussion_r966906318 still applies here and hasn't been resolved:

> Applying a transformation to IMU data requires transforming both the body and the world frames.

I still don't think transforming the world frame is correct here.

Reply:
I don't see that sentence in this document or in the previous discussion. I'm not sure what change or issue you have is :frowning:

Reply:
It is a few lines above, chapter Transformation: https://github.com/ros-infrastructure/rep/blob/a7e5e6a337a859f8ca789e5aa65067b508ef466d/rep-0145.rst#transformation

Reply:
The previous discussion:

@peci1

> I never understood the idea of this paragraph. If I transform both the sensor frame and the reference frame, isn't the resulting data transform always identity? I imagine applying a transform to IMU data is like taking the IMU and rotating its body. I.e. only changing the sensor frame transform regarding the same reference frame. How would I rotate ENU by 90 degrees yaw?

@mintar

> I agree this could be phrased better.

> Here's a relevant email by @paulbovbel (the original author of REP 145) that probably explains his thinking when he wrote that paragraph: https://groups.google.com/g/ros-sig-drivers/c/Fb4cxdRqjlU/m/MwS3uZDyfkoJ

> I suspect this paragraph is mixing two things, but I'm not 100% certain:

>     NED -> ENU conversion: You have to change body and world frames and the orientation.
>     transformation into a different body frame (e.g., from imu_link to base_link): You only have to change body frame and orientation; the world frame stays the same.